### PR TITLE
docs(readme): use a PR to bump a version

### PR DESCRIPTION
**Pull request by EtienneM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,12 +59,21 @@ Bump new version number in `CHANGELOG.md` and `README.md`.
...
+
+git switch --create release/${version}
 git add CHANGELOG.md README.md
-git commit -m "Bump v5.0.7"
-git tag v5.0.7
-git push origin master
-git push --tags
-gh release create v5.0.7
+git commit -m "Bump v${version}"
+git push --set-upstream origin release/${version}
```
Comment (ipfaze) [inline README.md:67]: question: is it a required command? I don't remember doing this one before using `gh pr create ...`

File: README.md [hunk 2]
```diff
@@ -59,12 +59,21 @@ Bump new version number in `CHANGELOG.md` and `README.md`.
...
+gh pr create --reviewer=EtienneM --title "$(git log -1 --pretty=%B)"
+```
+
+Once the pull request merged, you can tag the new release.
+
+```sh
+git tag v${version}
```
Comment (ipfaze) [inline README.md:74]: question: is it something that we can automate? Using GitHub Actions?

**Review by ipfaze — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (EtienneM) [reply README.md:74]: I don't think so. At least it's not how we do it on other open source projects.

File: README.md (hunk 1 above)
Comment (EtienneM) [reply README.md:67]: It's indeed not mandatory. `gh pr create` will do it if needed. I wanted it to be explicitly done in this procedure. That's how we documented it in other open source projects.

**Review by ipfaze — APPROVED** [review]
I agree with your answers, thanks! :pray: 
LGTM :rocket:

**Pull request merged**